### PR TITLE
Fix updating of motion comments for inmemory cache.

### PR DESCRIPTION
**Pull request by emanuelschuetze**
get_restricted_data() has to use deepcopy(full_data) instead of
full_data.copy().

Thanks to @ostcar and @normanjaeckel !

### CODE REVIEW

File: openslides/motions/access_permissions.py [hunk 1]
```diff
@@ -29,7 +29,8 @@ def get_restricted_data(self, full_data, user):
         if user.has_perm('motions.can_see_and_manage_comments') or not full_data.get('comments'):
             data = full_data
         else:
-            data = full_data.copy()
+            from copy import deepcopy
```
Comment (normanjaeckel) [inline openslides/motions/access_permissions.py]: Sorry. This import has to be at the top of the file. My hint yesterday was a little bit too short.

Comment (normanjaeckel) [inline openslides/motions/access_permissions.py]: Then you can merge it.

Comment (emanuelschuetze) [inline openslides/motions/access_permissions.py]: Done. Ready for merge.

**Comment by emanuelschuetze** [comment]
@normanjaeckel done. please approve and merge

**Pull request merged**